changelog/@unreleased/pr-2459.v2.yml

```diff
@@ -0,0 +1,5 @@
+type: improvement
+improvement:
+  description: Don't mock `ErrorDecoder` in `ConjureBodySerDeTest`
+  links:
+  - https://github.com/palantir/dialogue/pull/2459
```

Comment:
no need to add changelogs for changes that only update tests, users won't be impacted. Doesn't hurt anything, no need to revert this, just something for the future :-)

Reply:
Sounds good!